# 
# Translators:
# rldhont <rldhont@gmail.com>, 2018
# etrimaille <gustrimaille@yahoo.fr>, 2023
# 
msgid ""
msgstr ""
"Project-Id-Version: Lizmap Web Client 3.5 dataviz\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-03-21 14:22++0100\n"
"PO-Revision-Date: 2018-07-14 15:19+0000\n"
"Last-Translator: etrimaille <gustrimaille@yahoo.fr>, 2023\n"
"Language-Team: French (https://app.transifex.com/3liz-1/teams/62012/fr/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: fr\n"
"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % 1000000 == 0 ? 1 : 2;\n"

#: dataviz~dataviz.dock.subtitle
msgctxt "dataviz~dataviz.dock.subtitle"
msgid "Dataviz"
msgstr "Statistiques sur les données"

#: dataviz~dataviz.dock.title
msgctxt "dataviz~dataviz.dock.title"
msgid "Dataviz"
msgstr "Statistiques"

#: dataviz~dataviz.log.access_denied.detail
msgctxt "dataviz~dataviz.log.access_denied.detail"
msgid ""
"You have no right to access this ressource. Have you used the right login ?"
msgstr ""
"Vous n'avez pas les droits pour accéder à cette ressource. Avez-vous utilisé"
" les bons identifiants ?"

#: dataviz~dataviz.log.access_denied.title
msgctxt "dataviz~dataviz.log.access_denied.title"
msgid "Access denied"
msgstr "Accès refusé"

#: dataviz~dataviz.log.invalid_plot_type.detail
msgctxt "dataviz~dataviz.log.invalid_plot_type.detail"
msgid "The given plot type \"%s\" is invalid."
msgstr "Le type de graphique fourni \"%s\" est invalide."

#: dataviz~dataviz.log.invalid_plot_type.title
msgctxt "dataviz~dataviz.log.invalid_plot_type.title"
msgid "Invalid plot type"
msgstr "Type de graphique invalide"

#: dataviz~dataviz.log.layer_not_found.detail
msgctxt "dataviz~dataviz.log.layer_not_found.detail"
msgid ""
"The layer with the given ID \"%s\" cannot be found in the published QGIS "
"project."
msgstr ""
"La couche avec l'identifiant donné \"%s\" ne peut pas être trouvé dans le "
"projet QGIS."

#: dataviz~dataviz.log.layer_not_found.title
msgctxt "dataviz~dataviz.log.layer_not_found.title"
msgid "Layer not found"
msgstr "Couche non trouvée"

#: dataviz~dataviz.log.no_configuration.title
msgctxt "dataviz~dataviz.log.no_configuration.title"
msgid "There is no dataviz configuration for this project yet."
msgstr "Il n'y a pas encore de configuration de dataviz pour ce projet."

#: dataviz~dataviz.log.no_data.detail
msgctxt "dataviz~dataviz.log.no_data.detail"
msgid "The plot source data cannot be fetched from the vector layer \"%s\"."
msgstr ""
"Les données sources du graphique ne peuvent pas être récupérées pour la "
"couche vecteur \"%s\"."

#: dataviz~dataviz.log.no_data.title
msgctxt "dataviz~dataviz.log.no_data.title"
msgid "Plot data not available"
msgstr "Les données du graphique ne sont pas disponibles"

#: dataviz~dataviz.log.no_layers.detail
msgctxt "dataviz~dataviz.log.no_layers.detail"
msgid "The table of layers inside the dataviz configuration is empty."
msgstr "La table des couches dans la configuration de la dataviz est vide."

#: dataviz~dataviz.log.no_layers.title
msgctxt "dataviz~dataviz.log.no_layers.title"
msgid "Dataviz layers is empty"
msgstr "La couche dataviz est vide."

#: dataviz~dataviz.log.plot_configuration_not_found.detail
msgctxt "dataviz~dataviz.log.plot_configuration_not_found.detail"
msgid ""
"The plot configuration cannot be found or parsed by Lizmap dataviz API."
msgstr ""
"La configuration du graphique n'a pas été trouvée ou parsée par l'API "
"dataviz Lizmap."

#: dataviz~dataviz.log.plot_configuration_not_found.title
msgctxt "dataviz~dataviz.log.plot_configuration_not_found.title"
msgid "Plot configuration not found"
msgstr "Configuration du graphique non trouvée"

#: dataviz~dataviz.log.project_not_found.detail
msgctxt "dataviz~dataviz.log.project_not_found.detail"
msgid ""
"The QGIS project \"%s.qgs\" does not exist or is not available in the "
"repository \"%s\"."
msgstr ""
"Le projet QGIS \"%s.qgs\" n'existe pas ou n'est pas disponible dans le "
"répertoire \"%s\"."

#: dataviz~dataviz.log.project_not_found.title
msgctxt "dataviz~dataviz.log.project_not_found.title"
msgid "Lizmap project not found"
msgstr "Le projet Lizmap n'est pas trouvé."

#: dataviz~dataviz.log.repository_not_found.detail
msgctxt "dataviz~dataviz.log.repository_not_found.detail"
msgid "The Lizmap repository \"%s\" does not exist or is not available."
msgstr "Le répertoire Lizmap \"%s\" n'existe pas ou n'est pas disponible."

#: dataviz~dataviz.log.repository_not_found.title
msgctxt "dataviz~dataviz.log.repository_not_found.title"
msgid "Lizmap repository not found"
msgstr "Répertoire Lizmap non trouvé"

#: dataviz~dataviz.log.request_not_supported.detail
msgctxt "dataviz~dataviz.log.request_not_supported.detail"
msgid "The given request \"%s\" is not supported by Lizmap dataviz API."
msgstr "La requête \"%s\" n'est pas supportée par l'API dataviz Lizmap."

#: dataviz~dataviz.log.request_not_supported.title
msgctxt "dataviz~dataviz.log.request_not_supported.title"
msgid "Request is not supported"
msgstr "Requête non supportée"

#: dataviz~dataviz.log.wrong_credentials.detail
msgctxt "dataviz~dataviz.log.wrong_credentials.detail"
msgid "The given login and password are incorrect. Cannot authenticate."
msgstr ""
"Le nom d'utilisateur et le mot de passe ne sont pas corrects. Impossible de "
"s'authentifier."

#: dataviz~dataviz.log.wrong_credentials.title
msgctxt "dataviz~dataviz.log.wrong_credentials.title"
msgid "Wrong credentials"
msgstr "Identifiant incorrect"
